_posts/2022-06-02-survey-2022.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Strimzi survey: we want to hear from you!"
+date: 2021-06-02
+author: paolo_patierno
+---
+
+With the help of the CNCF, we're running another survey of the Strimzi community.
+The main purpose is to hear from Strimzi users in order to know how they are using Strimzi.
+This is really useful to the maintainers to make better decisions in terms of new features and define the roadmap of the project itself.
+
+It would be great if you could take the 5–10 minutes needed to answer the 10 questions.
```

Comment:
I know we have the link at the end, but I think we could afford to have a link here in the lede too.

```suggestion
It would be great if you could spare the 5–10 minutes needed to [take the survey](https://www.surveymonkey.com/r/Strimzi-2022).
```